descqa/SizeStellarMassLuminosity.py

```diff
@@ -37,20 +37,25 @@ def __init__(self, **kwargs):
         self.label_template = kwargs['label_template']
         self.fig_xlabel = kwargs['fig_xlabel']
         self.fig_ylabel = kwargs['fig_ylabel']
-
+        self.fig_subplot_row = kwargs['fig_subplot_row']
```

Comment:
Consider provide a default value in case the keyword wasn't provided in the yaml file. For example,
```python
self.fig_subplot_row = kwargs.get('fig_subplot_row', 2)
```
Same for nearby lines.

Reply:
I think you didn't get the right files

Reply:
Please include the correct config files in this PR.